### DISCUSSION TITLE
can i compile it for 820m nvidia driver

### DISCUSSION THREAD

**Discussion by vizvasrj** [NVIDIA/open-gpu-kernel-modules#605] [Q&A]
390.157 is only version that i can use but when i upgrade my linux debian system to newer version i cannot use older version driver for some reason how to compile it for nvidia 820m

**Comment by aaronp24** [comment]
Hi vizvasrj. The GeForce 820M is not supported by this driver repository. In addition, the 390 driver series is no longer supported. Please see the knowledge base article, _[Support timeframes for Unix legacy GPU releases](https://nvidia.custhelp.com/app/answers/detail/a_id/3142)_, for more information.
